packages/components/src/components/PipelineRuns/PipelineRuns.js

```diff
@@ -127,10 +127,13 @@ const PipelineRuns = ({
             {!pipelineName && (
               <StructuredListCell>
                 <Link
-                  to={createPipelineRunsByPipelineURL({
```

Comment:
See the cell above for how we handle this for the pipelineRun link: https://github.com/tektoncd/dashboard/pull/764/files#diff-2989be414084b3e10b098bac6149761fR125

We should do similar here, output an empty string instead of the `Link` component when `pipelineRefName` isn't available. This avoids rendering unnecessary DOM elements, and also avoid an accessibility issue where we'd have an invalid link with no URL and no text.